### DISCUSSION TITLE
Management URL of Tyk Gateway (OSS)

### DISCUSSION THREAD

**Discussion by anup-krai** [TykTechnologies/tyk-operator#277] [Q&A] [answered]
I am trying to install tyk-operator (oss) and have already installed Tyk Gateway(oss). There is a TYK_URL key which needs to be passed in the secret tyk-operator-conf.
Where can I find this URL? There is an example value given in the Installation page but that is for dashboard -

https://github.com/TykTechnologies/tyk-operator/blob/master/docs/installation/installation.md#tyk-operator-configuration

Thanks

**Comment by gernest** [comment, marked as answer]
It depends on how you configured tyk there is a `control_api_port` option  in tyk configuration . so if this is `9696` 

```
$TYK_URL=$TYK_HOST:9696
```

where `$TYK_HOST` is host where your tyk is running (if its on k8s then the dns to the tyk service)

**Reply by anup-krai** [reply]
We have created a TransportServer with virtual server IP and port which in-turn calling tyk service in K8. So do I need to give virtualServerAddress IP:9696 i.e. without the virtualServerPort?

**Reply by gernest** [reply]
If the TransportServer can also map/proxy `IP: 9696 ` to `TYK_SERVICE: 9696 ` then yes, providing you have a tyk service exposing `9696`

__simple solution__
Use the same port port that tyk is running on for api's and for control api. This is not recommended, do this if you just want to try out the operator.

Say your tyk configuration looks like this 

```json
    "listen_port": 8080,
    "control_api_port": 9696,
```

Change it to 
```json
    "listen_port": 8080,
    "control_api_port": 8080,
```

Now you just need to point to your TrasnportServer  url as `$TYK_URL` and it should work.
